schemas/EGA.assay.json

```diff
@@ -0,0 +1,327 @@
+        "title": "Object's IDs block",
+        "description": "Node containing the main identifiers of the object (e.g. alias, center_name...), inherited from the common definitions. #! We extend the core object (object_core_id) by adding a pattern check based on this schema's nature: an assay (by using EGA-assay-id-pattern)", 
+        "allOf": [
+          {
+            "title": "Inherited object_core_id object",
+            "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/object_core_id"
+          },
+          {
+            "title": "Check that assay's EGA ID (EGAR) is correct",
+            "properties": {
+              "ega_accession": {
+                "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/EGA-assay-id-pattern"
+              }
+            }
+          }
+        ]        
+      },
+
+      "schema_descriptor": {
+        "title": "Schema descriptor node",
+        "description": "Inherited schema descriptor node containing metadata about the schemas and standards used to create the JSON document itself.",
+        "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/schema_descriptor"
+      },
+
+      "object_title": {
+        "type": "string",
+        "title": "Title of the assay",
+        "description": "An informative assay title that should serve as an overview of the assay and differentiate it from others. This short text can be used to call out assay records in searches or in displays.",
+        "minLength": 1,
+        "examples": [ "Ilumina sequencing assay 3409 - Cancer genomics" ]
+      },
+
+      "object_description": {
+        "type": "string",
+        "title": "Description of the assay",
+        "description": "An in-depth description (e.g. used technology, sample groups, purpose...) of the assay. ",
+        "minLength": 1,
+        "examples": [ "Sequencing assay number 3409 of 4000. Sequenced through Illumina MiSeq to find SNPs of colorectal cancer samples..." ]
+      },
+
+      "assay_center": {
+        "type": "string",
+        "title": "Centername that performed the assay",
+        "description": "The name of the center (e.g. 'EBI-TEST') responsible for performing assay, if applicable, in case it's different from the center submitting metadata).",
+        "minLength": 1,
+        "examples": [ "EBI-TEST" ]
+      },
+
+      "assay_date": {
+        "type": "string",
+        "title": "Date of the assay",
+        "description": "ISO date (format YYYY-MM-DD - e.g. '2021-05-15') when the sequencing assay took place. If the protocols are long enough, the date shall be the day the assay concluded.",
+        "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/EGA-ISO-date-YYYY-MM-DD-pattern"
+      },
+
+      "sample_number": {
```

Comment:
Suggest removing sample_number, as discussed in previous comments

Reply:
done

Reply:
But just for the record, we shall always take into account the sample_grouping to count samples linked to other objects.